AirCasting/Extensions/DataBuilder.swift

```diff
@@ -0,0 +1,39 @@
+// Created by Lunar on 24/01/2022.
+//
+import Foundation
+
+// swiftlint:disable airCasting_date
+class DateBuilder {
+    static func getDate() -> Date {
```

Comment:
let's be explicit here and use appropriate names: `getDateMovedToUTCZone ` or `getFakeUTCDate`, idk. Something better. Also please add documentation for each function that is not self explanatory (add a comment with tripple slash `///` above function)